### PR TITLE
Repo: Add routing and necessary pages

### PR DESCRIPTION
**Pull request by cindykandie**
## This branch included the addition of multiple features
- I added routing between pages
- Adding the template for all the components and pages in the app
- Integrating the thoughts functionality from question to answer to update

### CODE REVIEW

File: client-side/public/thought-bubble.png
Comment (otsembo) [inline client-side/public/thought-bubble.png:1]: Kindly add all images into `public/assets/images`

File: client-side/src/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import MyThoughts from './pages/MyThoughts';
+import Favourites from './pages/Favourites';
+import AddMyThought from './pages/AddMyThought';
+import AboutApp from './pages/AboutApp';
+import Home from './pages/Home';
+
+const AppRoutes: React.FC = () => {
+  return (
+        <Routes>
+        <Route path="/" element={<Home />}/>
```
Comment (otsembo) [inline client-side/src/Routes.tsx:12]: Indent this portion for better visual appeal in your code.

**Review by otsembo — APPROVED** [review]
Good Job, on your initial PR.
Look at the few comments added, fix and then merge the PR.

File: client-side/public/thought-bubble.png
Comment (cindykandie) [reply client-side/public/thought-bubble.png:1]: I will, thank you!

File: client-side/src/Routes.tsx (hunk 1 above)
Comment (cindykandie) [reply client-side/src/Routes.tsx:12]: Sure thing! Thank you

**Pull request merged**